.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+Welcome to the Julia project!
+
+We use the GitHub issue tracker for bug reports and feature requests.
+If you have a question or are unsure if the behavior you're experiencing is a bug,
+please post to [our Discourse site](https://discourse.julialang.org).
+
+If you're submitting a bug report, be sure to include as much relevant information as
+possible, including a minimal reproducible example and the output of `versioninfo()`.
```

Comment:
I'm not sure it'd be all that valuable.  The template is plain-text, so there's a much bigger hurdle in getting the author to select the URL, copy it, paste it into the URL bar, and go to the page.  If we could actually make it a formatted link then I'd be in favor, but as it is I think it'd just end up making the template look noisier.